packages/server/lib/browsers/firefox.ts

```diff
@@ -335,20 +322,18 @@ const defaultPreferences = {
   'browser.helperApps.neverAsk.saveToDisk': downloadMimeTypes,
 }
 
-// CDP is deprecated in Firefox 129 and up.
+// CDP was deprecated in Firefox 129 and up and was removed in Firefox 141.
 // To enable BiDi (without CDP), we need to set
 //    remote.active-protocol=1
-// In order to enable CDP (without BiDi), we need to set
+// In order to enable CDP (without BiDi on firefox older than version 135), we need to set
```

Comment:
Should this comment just be updated to remove active-protocol 2 and 3 since we don't support them anymore?

Reply:
I wanted to keep some type of historical reference as to how to turn on CDP if needed. I think this comment needs some word smithing.